DOCKER.md

```
@@ -0,0 +1,307 @@
+### Permission issues with data directory
+
+```bash
+# Ensure proper ownership (rootless Docker uses your UID)
+chown -R $(id -u):$(id -g) ./data
+```
+
+### Bot can't connect to Discord
+
+1. Verify `.env` credentials are correct
+2. Check network connectivity:
+   ```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)